Data/SourceAssets/Localization/English/IngameTrainingInstructions.txt

```diff
@@ -1,14 +1,14 @@
-Enter the room right in front of you
-Stelle Dich unmittelbar vor die T�r.\Dr�cke �Enter
-Nimm den Schl�ssel
-Der Schl�ssel liegt nun im Inventar.\Die T�r l�sst sich nun �ffnen
-Ziehe den Hebel an der Wand\damit sich die T�r �ffnet
-In diesem Raum ist ein Geheimgang\hinter einer Wand
-Ziehe alle Hebel
-Bewege Dich schnell durchs Feuer\um wenig Schaden zu nehmen
-Nimm nun die Medizinflasche\um die Lebensenergie wieder aufzufrischen
-Wechsle mit F2 ins Inventar-Fenster\und bewege die Medizinflasche auf\das Symbol �Messer und Gabel
-Nimm alle Waffen aus diesem Raum.\Du wirst sie brauchen...
-Klettere die Leiter mit\der Sprungtaste hoch.
-Spring ins Wasser.
-Ziehe unter Wasser den Hebel\und tauche zum Ausgang.
+Enter the room right in front of you.
+Stand right in front of the door.\Press �Return
```

Comment:
Not correct in the original version, but please fix: Full stop after "Press Return"